lib/IRGen/GenStruct.cpp

```diff
@@ -61,6 +63,47 @@ static StructTypeInfoKind getStructTypeInfoKind(const TypeInfo &type) {
   return (StructTypeInfoKind) type.getSubclassKind();
 }
 
+clang::CXXDestructorDecl *swift::irgen::getDestructor(SILType type) {
+  DestructorDecl *destructor = nullptr;
+  if (auto structDecl = type.getStructOrBoundGenericStruct()) {
+    for (auto member : structDecl->getMembers()) {
+      if (auto dest = dyn_cast<DestructorDecl>(member)) {
+        destructor = dest;
+        break;
+      }
+    }
+  } else if (auto *classDecl = type.getClassOrBoundGenericClass()) {
```

Comment:
We always import C++ classes as Swift structs (not classes), so this is superfluous.